packages/auth/express/auth.js

```diff
@@ -67,13 +72,26 @@ exports.configure = ({
     const user = transformUser(
       await Users.findOne({id})
     )
+
     if (!user) {
       return next('user not found!')
     }
+
     next(null, user)
   })
 
   // Initialise Passport
   server.use(passport.initialize())
   server.use(passport.session())
+
+  server.use(function (req, res, next) {
+    // Check if a user has more than one and let session expire after a shorter period of time
+    if (req.user && userIsInRoles(req.user, specialRoles)) {
+      req.session.cookie.expires =
```

Comment:
I think setting maxAge is enough:
- https://github.com/expressjs/session#rolling
- https://github.com/voxpelli/node-connect-pg-simple/blob/master/index.js#L207

P.S. stupid prettier making two lines out of this :1st_place_medal:

Reply:
That worked. To set `maxAge` is enough.